src/components/MoneyRequestConfirmationList.js

```diff
@@ -388,6 +401,16 @@ function MoneyRequestConfirmationList(props) {
                         onPress={() => Navigation.navigate(ROUTES.getMoneyRequestMerchantRoute(props.iouType, props.reportID))}
                         disabled={didConfirm || props.isReadOnly || !isTypeRequest}
                     />
+                    {props.policyCategories !== null && (
```

Comment:
Should we do an empty check here instead? Like is it possible that it could be an empty array, for example?

Reply:
Done - b0da998742f4ba73d399dcb62471678ed38151de

Reply:
@puneetlath, @yuwenmemon, I have to mention that the `policyCategories` collection is an object, not an array. Also, when there are no categories for a policy, **Onyx** does not pass anything to props. At this stage the default props work.
Anyway, I decided to:
1. Use an empty object as a default prop.
2. Check via `isEmpty`.
Does it make sense now?